vulnerabilities/importers/safety_db.py

```diff
@@ -137,21 +138,11 @@ def updated_advisories(self) -> Set[Advisory]:
 
         return self.batch_advisories(advisories)
 
-    def create_etag(self, url):
-        etag = requests.head(url).headers.get('ETag')
-        if not etag:
-            # Kind of inaccurate to return True since etag is
-            # not created
-            return True
-        elif url in self.config.etags:
-            if self.config.etags[url] == etag:
-                return False
-        self.config.etags[url] = etag
-        return True
-
 
 def categorize_versions(
-    package_name: str, all_versions: Set[str], version_specs: Iterable[str],
+    package_name: str,
+    all_versions: Set[str],
+    version_specs: Iterable[str],
```

Comment:
consistency nit: Do we call these specs or ranges?

Reply:
I'm renaming that to version_ranges. The feeds call it specs, which is probably why we called it specs here.